src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala

```diff
@@ -69,6 +69,17 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues(): Unit = {
+    val isEmpty = List(1).size == 0
+    val b = if (true) true else false
+    var TWO = 1 + 1
+    //test
```

Comment:
MINOR: Space after the start of the comment ([more](http://localhost/coding_rules?open=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker&rule_key=sonar-scala-scalastyle%3Aorg.scalastyle.scalariform.SpaceAfterCommentStartChecker))